### PR TITLE
added navbar and hero image

### PR DESCRIPTION
**Pull request by seanmckay94**

### CODE REVIEW

File: assets/css/style.css [hunk 1]
```diff
@@ -0,0 +1,125 @@
+* {
```
Comment (rachel-brain) [inline assets/css/style.css]: can you please add this to ::before, ::after too.

File: assets/css/style.css [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+  box-sizing: border-box;
+  margin: 0;
```
Comment (rachel-brain) [inline assets/css/style.css]: remove margin 0.

File: assets/css/style.css [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+  font-family: 'Bellota Text', 'Verdana', Arial, sans-serif;
+}
+
+/* Color scheme sourced from https://www.color-hex.com/color-palette/107368 70s colors color palette */
+:root {
```
Comment (rachel-brain) [inline assets/css/style.css:7]: add more descriptive names to the variables. what is colora, b and c?

File: assets/css/style.css [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+  --colora: #00a0d9;
+  --colorb: #b800a6;
+  --colorc: #68a549;
+  --colord: #ff4ea6;
+  --colore: #fdcd00;
+}
+
+body {
+  font-size: 16px;
```
Comment (rachel-brain) [inline assets/css/style.css]: default is 16px, remove

File: assets/css/style.css [hunk 5]
```diff
@@ -0,0 +1,125 @@
...
+  line-height: 1;
```
Comment (rachel-brain) [inline assets/css/style.css]: default value is 1 so remove please

File: assets/css/style.css [hunk 6]
```diff
@@ -0,0 +1,125 @@
...
+  background: linear-gradient(rgba(0, 0, 0, 0.5), rgba(0, 0, 0, 0.5));
+}
+
+.btn {
+  font-size: 20px;
+  border: var(--colora);
+  border-radius: 10px;
+  border: 15px;
```
Comment (rachel-brain) [inline assets/css/style.css]: you can simplify this as:

```
border: 15px solid var(--colora);
border-radius: 10px 
```

File: assets/css/style.css [hunk 7]
```diff
@@ -0,0 +1,125 @@
...
+  cursor: pointer;
+  background-color: var(--colora);
+}
+  
+.header {
+  padding: 0px;
+  background-color: var(--colord);
+  color: var(--colora);
+  /* position: fixed; */
+  width: 100%;
```
Comment (rachel-brain) [inline assets/css/style.css]: default is width 100%;

File: assets/css/style.css [hunk 8]
```diff
@@ -0,0 +1,125 @@
...
+  background-size: 100%;
+  object-fit: cover;
+}
+
+.header-container {
+    padding: 0px;
+    /* background-color: var(--colord); */
+    color: var(--colora);
+    /* position: fixed; */
+    width: 100%;
```
Comment (rachel-brain) [inline assets/css/style.css]: same as above

File: assets/css/style.css [hunk 9]
```diff
@@ -0,0 +1,125 @@
...
+    height: 100%;
+}
+
+.header h1 {
+  display: inline-block;
```
Comment (rachel-brain) [inline assets/css/style.css]: remove

File: assets/css/style.css [hunk 10]
```diff
@@ -0,0 +1,125 @@
...
+
+#header-icon {
+  color: black;
+}
+
+.header nav {
+  padding-top: 50px;
+  padding-right: 40px;
+  margin-right: 20px;
+  float: right;
```
Comment (rachel-brain) [inline assets/css/style.css]: remove floats and use flex

File: assets/css/style.css [hunk 11]
```diff
@@ -0,0 +1,125 @@
...
+.header nav ul {
+  list-style-type: none;
+}
+
+.header nav ul li {
+  display: inline-block;
+  margin-left: 25px;
+}
+
+
```
Comment (rachel-brain) [inline assets/css/style.css]: remove white space

File: assets/css/style.css [hunk 12]
```diff
@@ -0,0 +1,125 @@
...
+    margin-bottom: 20px;
+    font-size: 36px;
+}
+
+
+
+
+.footer {
+  padding: 30px;
+  clear: both;
```
Comment (rachel-brain) [inline assets/css/style.css]: your footer doesn''t have floats, remove clear

File: index.html [hunk 13]
```diff
@@ -5,71 +5,70 @@
...
+            <div class="topnav">
+            <nav>
+                <ul><!-- add internal nav links here -->
+                    <li><button class="btn"><a href="#search-bar"><i class="fas fa-search"></i> Find a gig</a></button></li>
+                    <li><button class="btn"><a href="#calendar"><i class="far fa-calendar-alt"></i> Calendar</a></button></li>
+                    <li><button class="btn"><a href="#upcoming-events"><i class="fas fa-exclamation-circle"></i> Upcoming events</a></button></li>
+                </ul>
+            </nav>
+            </div>
+            </div>
```
Comment (rachel-brain) [inline index.html]: this div should be on the line below, belongs to the hero text div

File: index.html [hunk 14]
```diff
@@ -5,71 +5,70 @@
...
+        </header>
+    </div>
 
+<!-- Add a hero image? Crowds & a stage? -->
     <img class="hero" alt=""></img>
 
     <main class="content">
-        <section class="search-bar">
-            <img src="???" class="float-left" alt="Photo of ???" />
+        <section class="search-bar" id="search-bar">
```
Comment (rachel-brain) [inline index.html]: you should use a article here.

File: index.html [hunk 15]
```diff
@@ -5,71 +5,70 @@
...
-            <div data-tockify-component="mini" data-tockify-calendar="cool.app"></div>
+    <aside class="information">
+        <section class="calendar" id="calendar">
+            <h3>Calendar link - default to current month?</h3>
+            <p><!-- Add calendar here (API?) --></p>
         </section>
 
         <section class="map">
-            <h3>Choose your location</h3>
-            <iframe src="https://www.google.com/maps/embed?pb=!1m14!1m12!1m3!1d50423.10888319078!2d144.95594994646723!3d-37.82648314313222!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!5e0!3m2!1sen!2sau!4v1617275375044!5m2!1sen!2sau"
```
Comment (rachel-brain) [inline index.html:61]: re-add this iframe as its google maps

**Review by rachel-brain — CHANGES_REQUESTED** [review]
Do a better job

File: assets/css/style.css (hunk 1 above)
Comment (seanmckay94) [reply assets/css/style.css]:
```
*, *::before, *::after {
    box-sizing: border-box;
}
```

**Review by JWCoad — APPROVED** [review]

**Review by JWCoad — APPROVED** [review]
Seem's good

**Review by rachel-brain — APPROVED** [review]

**Pull request merged**